index.js

```diff
@@ -25,11 +24,10 @@ function generate(event){
     sSource = sSource.replace("1x", "x");
     solutionElement.src = sSource;
 
-
     solutionElement.style.visibility = "hidden";
 }
 
-function solution(event){
+function solve(event){
```

Comment:
Arguably should be called solution, or even showSolution, as this function doesn't solve anything, it just shows the already existing solution.

Reply:
Arguably you are wrong.